libs/sdk-ui-dashboard/src/model/commandHandlers/filterContext/attributeFilter/addAttributeFilterHandler.ts

```diff
@@ -82,9 +84,84 @@ export function* addAttributeFilterHandler(
 
     invariant(addedFilter, "Inconsistent state in attributeFilterAddCommandHandler");
 
+    yield put(
+        filterContextActions.updateFiltersToIndexMappingOnFilterAdded(
```

Comment:
Can the `filterContextActions.updateFiltersToIndexMappingOnFilterAdded` and `filterContextActions.updateConnectingAttributesOnFilterAdded` functions maybe be combined somehow? They should always be called together right, so it might be better to merge them to avoid inconsistencies in the store in case someone calls only one of them.

Reply:
Both adding and deleting actions are merged and are called together.